xwiki-platform-core/xwiki-platform-ldap/xwiki-platform-ldap-authenticator/src/main/java/com/xpn/xwiki/plugin/ldap/XWikiLDAPUtils.java

```diff
@@ -686,7 +716,12 @@ public boolean getGroupMembersSearchResult(LDAPSearchResults result, Map<String,
 
                     resultEntry = result.hasMore() ? result.next() : null;
                 } catch (LDAPException e) {
-                    LOGGER.debug("Failed to get group members", e);
+                    if(LOGGER.isInfoEnabled()){
+                        LOGGER.info("Failed to get group members", e);
+                        LOGGER.debug("Result Code {}",e.getResultCode());
+                        LOGGER.debug("LDAP server Message {}",e.getLDAPErrorMessage());
+                        LOGGER.debug("Matched DN {}",e.getMatchedDN());
```

Comment:
Why all the code duplication instead of using an utility method for those 3 debug messages?